gemfiles/rails_6.1.gemfile

```diff
@@ -8,10 +10,10 @@ group :development do
   gem "csv"
   gem "nokogiri"
   gem "pry-byebug"
+  gem "rails", "~> 6.1.0"
```

Comment:
Why not drop support for EOL Rails in v2? https://endoflife.date/rails

Reply:
I have no reason not too, but you are probably right.